apps/website/docs/setup/_guilded-setup.mdx

```diff
@@ -1,9 +1,12 @@
 ## Guilded
 
-Codeize is gonna write this part cause I have no fucking clue
+To get your Guilded API key
 
-so ttue
+First, head to your server that you want to be "home base" for your bot.
+Go into the drop down menu -> Manage bots, and in the window that opens click "Create Bot"
 
-wedfuohwefuohefwujihwerfhuiqwdefedhuoqwqwedfhuiohio
+### Your bots Token
+After the bot is created go into the API tab and click "Generate Token" this will show you your api key that wont get shown again.
 
-vrooom broooom browerfujniowerfhujniowefbjnkpwerfgophjwerfgnkertgh juiopw4th uw34r ioq23r jioqwer uioqwe uiowert jio
+### Adding your bot to other Guilds
+After generateing the token the "Publish bot" will become active and once toggled on your bot can be added to other servers but only API actions will work any flows only work on the server the bot was created in
```

Comment:
```suggestion
After generating the token, the "Publish bot" button will become active. Once toggled on your bot, it can be added to other servers, but only API actions will work. Any flows only work on the server the bot was created in.
```